oaknational/Oak-Web-Application
# feat: prelaunch changes

## Description
- Update the tab background colour to mint
- Update the main copy
    - from School support: https://docs.google.com/document/d/16k8Qimvf6XBhtrVA-0b_z248u5JMRR-McK0SdAobpvk/edit
- Update image in top banner to this one: (with the same black border as on the other homepage tabs)
- Make the AI squad the default homepage

## How to test

1. Go to https://deploy-preview-2750--oak-web-application.netlify.app/
2. aila should be the default tab
3. click on the other tabs and you should still be able to navigate to them

## Screenshots

How it used to look (delete if n/a):
<img width="1435" alt="image" src="https://github.com/user-attachments/assets/5d8c3f3d-4df1-4111-b538-e6f3767a9ef0">


How it should now look:
![image](https://github.com/user-attachments/assets/8e427e01-397c-477c-a7ad-5ca936d7e7a4)



### Known issues
- the image has a thinner border than the images on the other tabs. This is because on those images the border is in the static image itself. I am contacting marketing to fix this issue directly in sanity


### Acceptance Criteria

- [x]  Update tab background colour to mint (#BEF2BD)
- [x]  Update the main copy
    
    **Your lesson prep, transformed with AI**
    
    Try our free, AI-powered lesson assistant: Aila. Whether it’s creating bespoke resources or helping you tailor content to your class, Aila can help speed things along.
    
- [x]  Update the image in the top banner (attached above). Image position/size/stying as per the Teaching resources tab.
- [x]  Make the AI tab the default homepage
